tools/dreport.d/plugins.d/journalpid

```diff
@@ -1,20 +1,13 @@
 #!/bin/bash
 #
-# config: 134 1
-# @brief: Collect user initialized PID specific journal log information.
+# config: 13 1
```

Comment:
Cannot we use https://github.com/openbmc/phosphor-debug-collector/blob/master/tools/dreport.d/plugins.d/journalpretty? looks like we are collecting full journal traces for all types of the BMC dump, right? any specific reason?

Reply:
as per discussion in SOS, and in openbmc-general

Reply:
> Journal logs in core dumps and internal failures: Just grab the full journal

I think you are collecting full traces as discussed but, both plugins collect full traces so, cannot use only one by adding the config `13` into https://github.com/ibm-openbmc/phosphor-debug-collector/blob/1020/tools/dreport.d/plugins.d/journalpretty#L3?

and also we will miss entries if the application is added because we are using the `--no-pager`.